src/scripts/inline_bin.ts

```diff
@@ -0,0 +1,20 @@
+import fs from "fs";
+import path from "path";
+
+// eslint-disable-next-line prefer-const
+let [inputPath, outputPath = ""]: string[] = process.argv.slice(2);
+
+if (outputPath === "") {
+  const inputDir = path.dirname(inputPath);
+  const inputBase = path.basename(inputPath);
+  outputPath = path.join(inputDir, inputBase + ".js");
+}
+
+// TODO: something slightly fancier
+// fs.createReadStream(inputPath);
+const data = fs.readFileSync(inputPath);
+const js_contents = `
+import { Buffer } from "buffer";
```

Comment:
Is the buffer library already used for some other purpose or only imported here to convert to base64?

Reply:
Just for base64 decoding. Is there a more browser-idiomatic way you're aware of?

Reply:
We've been using this library https://www.npmjs.com/package/uint8arrays

Reply:
Gotcha, my thinking was that Buffer was already available via the webpack polyfill. Looking closer, I'm noticing that maybe all but [one usage of Buffer](https://github.com/ChainSafe/blst-ts/blob/bc9f2a46059bc437e1e045b086f8269d00c1343c/src/lib.ts#L303) is in tests.